job-runner/api/api.go

```diff
@@ -30,6 +30,9 @@ var (
 	ErrDuplicateTraverser = errors.New("traverser already exists")
 	ErrTraverserNotFound  = errors.New("traverser not found")
 	ErrInvalidTraverser   = errors.New("traverser found, but type is invalid")
+
+	// Error when Job Runner is shutting down and not starting new job chains
+	ErrNoNewJobChains = errors.New("Job Runner is shutting down - no new job chains are being started")
```

Comment:
I think `ErrShuttingDown` is more clear, since that's what's happening. Not accepting new job chains is a side-effecting of shutting down.